File: src/JavaScript/easy/string/problem1967.js
/**
 * @param {string[]} patterns
 * @param {string} word
 * @return {number}
 */

var numOfStrings = function(patterns, word) {
    let i = 0
    patterns.forEach(pattern => {
        if(word.includes(pattern)){
            i++
        }
    })
    return i
};


// let patterns = ["a","abc","bc","d"], word = "abc"
let patterns = ["a","b","c"], word = "aaaaabbbbb"

numOfStrings(patterns, word)

File: src/Python/easy/string/problem1967.py
from typing import List

class Solution:
    def numOfStrings(self, patterns: List[str], word: str) -> int:
        i = 0
        for pattern in patterns:
            if pattern in word:
                i += 1
        return i

patterns = ["a","abc","bc","d"]; word = "abc"

solution = Solution()
solution.numOfStrings(patterns, word)